packages/ui/src/Card.module.scss

```diff
@@ -0,0 +1,36 @@
+@use '../styles/constants' as c;
+
+.wrapper {
+  padding: c.$grid * 5;
+  box-shadow: 0 c.$cardShadowSize c.$cardShadowBlur 0 c.$colorShadow;
+  border-radius: c.$borderRadius;
+  background: c.$colorNeutralWhite;
+
+  h3.heading {
+    @include c.typographyH3();
+
+    display: flex;
+    align-items: center;
+    padding: 0;
```

Comment:
Is there a default padding on heading elements?

Reply:
Updated 👍